app/src/androidTest/java/com/swent/assos/SuperTest.kt

```diff
@@ -0,0 +1,30 @@
+package com.swent.assos
+
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import com.kaspersky.components.composesupport.config.withComposeSupport
+import com.kaspersky.kaspresso.kaspresso.Kaspresso
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import com.swent.assos.model.navigation.NavigationActions
+import dagger.hilt.android.testing.HiltAndroidRule
+import io.mockk.impl.annotations.RelaxedMockK
+import io.mockk.junit4.MockKRule
+import org.junit.Before
+import org.junit.Rule
+
+open class SuperTest : TestCase(kaspressoBuilder = Kaspresso.Builder.withComposeSupport()) {
```

Comment:
Good idea, it will allow us to avoid duplicating the same code everywhere to setup the tests